styles/styles.css

```diff
@@ -303,9 +303,17 @@ main > .section > .default-content-wrapper > p {
   margin: auto;
   max-width: 60%;
   text-align: left;
+  margin-top: 5vh;
```

Comment:
Why would you do this? This means that the distance between the autoblogblock changes based on browser height. Make it a PX value that matches the live site.

Reply:
That was done just to make things adjustable as per the device size but yes - height doesn't matter much. Done with the changes.